tests/unit/ForeignJurisdiction.spec.ts

```diff
@@ -29,80 +29,124 @@ describe('ForeignJurisdiction', () => {
       })
     const vm: any = wrapper.vm
 
-    wrapper.find('#country-selector').trigger('click')
-    await Vue.nextTick()
-    wrapper.find('.menuable__content__active').findAll('.v-list-item').at(123).trigger('click')
+    await wrapper.find('#country-selector').trigger('click')
+    wrapper.findAll('.v-list-item').at(123).trigger('click')
 
     expect(vm.selectedCountry).toEqual({code: 'LB', name: 'Lebanon'})
     expect(vm.selectedRegion).toEqual({})
```

Comment:
I'm wondering if this should expect null? (I know it depends on how you initialized this variable.)

I'm thinking ahead to what this value will look like in the store and how we will save it to the filing object. Maybe this will work?
```
filing.consentContinuationOut.foreignJurisdiction.region = this.getSelectedRegion.code || undefined
```

Reply:
This is exactly what I had in mind. It should work!

Shall I change the test? The test would fail since it's expecting `{}`. I'd have to change the initialization of the object. I think initializing to empty object should be OK though.

Reply:
Not sure. Thinking ahead is risky and especially on a Friday.

It may help to try implementing parts of 16528 (ie, save/resume) and see what works best.

Reply:
Agreed. Alright, sounds good.